R/sensitivity-analysis-run-options.R

```diff
@@ -17,8 +17,13 @@ SensitivityAnalysisRunOptions <- R6::R6Class(
                               showProgress = ospsuiteEnv$showProgress) {
       ref <- rClr::clrNew("OSPSuite.R.Domain.SensitivityAnalysisRunOptions")
       super$initialize(ref)
-      self$numberOfCoresToUse <- numberOfCoresToUse
-      self$showProgress <- showProgress
+
+      if (!is.null(numberOfCoresToUse)) {

```

Comment:
@abdullahhamadeh This weird check in the code should not be required anymore. .NET does not like the NULL symbol it seems that we have to wrap that into a if condition

Reply:
OK I will remove the check in reporting engine, so now a NULL can be passed to SensitivityAnalysisRunOptions initializer for numberOfCoresToUse